openpype/hosts/maya/plugins/publish/validate_animated_reference.py

```diff
@@ -0,0 +1,63 @@
+import pyblish.api
+from openpype.pipeline.publish import (
+    PublishValidationError,
+    ValidateContentsOrder
+)
+from maya import cmds
+
+
+class ValidateAnimatedReferenceRig(pyblish.api.InstancePlugin):
+    """
+        Validate all the nodes underneath skeleton_Anim_SET
+        should be reference nodes
+    """
```

Comment:
```suggestion
    """Validate all nodes in skeletonAnim_SET are referenced"""
```

Also make sure you always name the sets like you create them in create rig for clarity:
```python
        skeleton = cmds.sets(
            name=subset_name + "_skeletonAnim_SET", empty=True)
        skeleton_mesh = cmds.sets(
            name=subset_name + "_skeletonMesh_SET", empty=True)
```

So this should be `skeletonAnim_SET` instead of `skeleton_Anim_SET`. The same goes for all the logging messages below.